src/styles/blocks/contact-us.scss

```
@@ -0,0 +1,82 @@
+.contact-us {
+  &__content {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    column-gap: 20px;
+    @include onTablet {
+      display: grid;
+      grid-template-columns: repeat($column-table-mobile, 1fr);
+    }
+    @include onDesktop {
+      display: grid;
+      grid-template-columns: repeat($column-table-desktop, 1fr);
+      column-gap: 24px;
+    }
+  }
+
+  &__addres-one {
+    @include onTablet {
+      grid-column: 1/3;
+    }
+    @include onDesktop {
+      grid-column: 1/4;
+    }
+  }
+
+  &__addres-two {
+    @include onTablet {
+      grid-column: 1/3;
+    }
+    @include onDesktop {
+      grid-column: 4/8;
+    }
+  }
+
+  &__form-place {
+    @include onTablet {
+      grid-column: 4/7;
+      grid-row: 1/3;
```

```suggestion
      grid-column: 4/7;
      grid-row: 1/3;
```
in next header try to use loop for grid and add all this styles by class